src/pkg/packaging/deb/package.targets

```diff
@@ -195,6 +195,7 @@
   </Target>
 
   <Target Name="GenerateSharedFrameworkDeb">
+  
```

Comment:
unnecessary change

Reply:
Are you referring newline ? Other targets too have a newline, following the same pattern here.

Reply:
4 out of the 5 existing targets in this file don't start with a newline.

Reply:
You are right. I noticed just the one with the extra newline. Will fix this one and TestDebuild target.